### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LProulx86**
GitHub Issue: #

## Proposed Changes
update readme file

## Checklist

Please check if your PR fulfills the following requirements:

- [ ] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [ ] Contains **NO** breaking changes
- [ ] Updated the Changelog
- [ ] Associated with an issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,20 +22,22 @@ The following is the template for the final README.md file:
...
 
-{Instructions to quickly get started using the project: pre-requisites, packages
-to install, sample code, etc.}
+- Clone project.
+- For seeing sample, build and install app with VS on the desire device (Android, IOS, or UWP)
 
 ## Features
 
-{More details/listing of features of the project}
+- DynamicMap is an interactive map for Android, IOS or UWP.
```
Comment (carlh98) [inline README.md:38]: Add

**Review by carlh98 — CHANGES_REQUESTED** [review]
Add more details for the feature

File: README.md [hunk 2]
```diff
@@ -1,41 +1,134 @@
-# Open Source Project Template
+﻿# Cartography Refactor
```
Comment (carlh98) [inline README.md:1]: Should not have mention of the refactoring in the file

**Review by carlh98 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1,41 +1,134 @@
...
+- In your ViewModel :
+```
+using Cartography.DynamicMap
+```
+- Implement IDynamicMapComponent to your ViewModel
+- Set Initial Value to your ViewPort.
+- Add in your Page
+#### UWP
+```
+<win:Grid>
```
Comment (carlh98) [inline README.md:24]: We need to keep using win: at the moment?

File: README.md [hunk 4]
```diff
@@ -1,41 +1,134 @@
...
+   - `IOS Map` : Show "Apple" Map on screen available only on IOS(native)
+   - `Bing Map` : Show Bing Map on screen available only on UWP(native)
+
+2. `Show user location` : Show the user location on the map.
+
+3. `Show Pushpin` : Show Pushpin (marker) on the map.
+   - `Filter Pushpin`
+   - `Add Pushpin`
+   - `Remove Pushpin`
+   - `Customize pushpin` : Change apperance of pushpin.
```
Comment (carlh98) [inline README.md:101]: appearance

**Review by carlh98 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (LProulx86) [reply README.md:24]: Right now yes, but we want to change it.

File: README.md [hunk 5]
```diff
@@ -1,41 +1,134 @@
...
+   - `Show POI` : IOS only: show Point Of Interest. eg: Tour Eiffel.
+
+5. `Follow User`
+   - `Start follow user`
+   - `Stop follow user` : Can detect if dragging or on button press.
+   
+### StaticMap
+1. `Show Map`: Show a map on screen without interraction possible.
+   - `Google Map` : Show Google Map on screen available for UWP, IOS, Android(native)
+   - `IOS Map` : Show "Apple" Map on screen available only on IOS(native)
```
Comment (carlh98) [inline README.md:124]: You can remove the "" around Apple

File: README.md [hunk 6]
```diff
@@ -1,41 +1,134 @@
...
+            ));
+```
 
 ## Features
 
-{More details/listing of features of the project}
+### DynamicMap
+1. `Show Map` : Show a interactive map on screen
+   - `Google Map` : Show Google Map on screen available for UWP, IOS, Android(native)
+   - `IOS Map` : Show "Apple" Map on screen available only on IOS(native)
```
Comment (carlh98) [inline README.md:92]: You can remove the "" around Apple

File: README.md [hunk 7]
```diff
@@ -1,41 +1,134 @@
...
+- Add Cartography.StaticMap NuGet package to your project.
+- In your ViewModel :
+```
+using Cartography.StaticMap
+```
+- Implement IStaticMapComponent to your ViewModel
+- Set Initial Value to your ViewPort and MapSize.
+- Add in your Page 
+```
+<staticmap:StaticMapControl MapViewPort="{Binding MapViewPort}"
```
Comment (carlh98) [inline README.md]: Replace to ViewPort

**Pull request merged**